src/main/java/fr/eseo/tetris/Main.java

```diff
@@ -0,0 +1,12 @@
+package fr.eseo.tetris;
+
+public class Main {
+    public static void main(String[] args) {
+        Piece piece = new Piece(Tetromino.T);
+        piece.printShape();
+        System.out.println("\nRotation:");
+        piece.rotate();
+        piece.printShape();
```

Comment:
J'aime pas, c'est mieux comme ca : 
```suggestion
        Piece piece = new Piece(Tetromino.T);
```